sycl/doc/extensions/experimental/sycl_ext_oneapi_bindless_images.asciidoc

```diff
@@ -194,8 +194,11 @@ enum class image_channel_type : /* unspecified */ {
 enum class image_type : /* unspecified */ {
   standard,
   mipmap,
+<<<<<<< HEAD
+=======
   array,
   interop,
+>>>>>>> sycl
```

Comment:
looks like you committed some conflict markers here

Reply:
Thanks for pointing this out. Fixed.